nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/controller-service/controller-services.component.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="p-4 flex flex-col h-screen justify-between gap-y-5">
+    <div class="flex justify-between">
+        <h3 class="text-xl bold controller-services-header">Controller Services</h3>
+        <button class="nifi-button" [routerLink]="['/']">
+            <i class="fa fa-times"></i>
+        </button>
+    </div>
+    <div class="flex-1 flex flex-col">
+        <ng-container *ngIf="serviceState$ | async; let serviceState">
+            <div *ngIf="isInitialLoading(serviceState); else loaded">
+                <ngx-skeleton-loader count="3"></ngx-skeleton-loader>
+            </div>
+            <ng-template #loaded>
+                <div class="flex flex-col h-full gap-y-2">
+                    <div class="flex justify-between">
+                        <breadcrumbs
+                            [entity]="serviceState.breadcrumb"
+                            [currentProcessGroupId]="serviceState.processGroupId"></breadcrumbs>
```

Comment:
probably for a future discussion, but i was thinking that based on the context this breadcrumb is used in... one could argue that it shouldn't navigate you to the canvas with the selected pg loaded but rather to the controller services listing for the selected breadcrumb pg.